internal/cli/atlas/quickstart/access_list_setup.go

```diff
@@ -52,11 +45,17 @@ func (opts *Opts) askAccessListOptions() error {
 	if publicIP != "" {
 		message = fmt.Sprintf(" [Press Enter to use your public IP address '%s']", publicIP)
 	}
-	return survey.AskOne(
+	err := survey.AskOne(
```

Comment:
if this errors shouldn't you exit?

Reply:
thanks, yes! I can make line 54 `if err == nil && opts.IPAddressesResponse != "" {`